pkgs/applications/editors/imhex/default.nix

```diff
@@ -77,6 +77,7 @@ gcc12Stdenv.mkDerivation rec {
   postInstall = ''
     ln -s $out/share/imhex/plugins $out/bin/
```

Comment:
The built-in plugin is found even when the symlink is not there. But I'm not sure if the files from `patterns_src` are correctly loaded (maybe they should go somewhere in `$out/lib/imhex` instead of `$out/share/imhex`).

```suggestion
```

Reply:
Also they should not be under $out/bin